.circleci/config.yml

```diff
@@ -110,6 +117,11 @@ references:
       command: |
         TESTS=$(circleci tests glob "spec/{requests,swagger}/**/*_spec.rb" | grep -e rswag -e swagger | circleci tests split )
         bundle exec rspec ${TESTS} --format Rswag::Specs::SwaggerFormatter --order defined
+  _run-wiremock: &run-wiremock
+    run:
+      name: Run prison-api wiremock
+      command: java -jar spec/wiremock/wiremock-standalone.jar --port 8888 --verbose --root-dir spec/wiremock/prison-api
+      background: true
```

Comment:
This could also be covered by the test-executor image, if there is one.  Or next best thing the base java docker image where just  pull the jar and run it.